Godeps/Godeps.json

```diff
@@ -177,6 +197,11 @@
 			"Rev": "092cba3727bb9b4a2f0e922cd6c0f93ea270e363"
 		},
 		{
+			"ImportPath": "github.com/fsnotify/fsnotify",
+			"Comment": "v1.3.1-1-gf12c623",
+			"Rev": "f12c6236fe7b5cf6bcf30e5935d08cb079d78334"
```

Comment:
Let's make sure we are using the same version with cri-o to avoid unnecessary conflict in the future.
7d7316ed6e1ed2de075aab8dfc76de5d158d66e1